versioned_docs/version-2.7/integrations-in-rancher/cloud-marketplace/aws-marketplace-payg-integration/installing-rancher-prime.md

```diff
@@ -0,0 +1,160 @@
+  ```
+
+**Response:**
+
+  ```shell
+  NAMESPACE                           NAME                          READY   UP-TO-DATE   AVAILABLE   AGE
+  cattle-csp-billing-adapter-system   csp-rancher-usage-operator    1/1     1            1           30m
+  cattle-csp-billing-adapter-system   rancher-csp-billing-adapter   1/1     1            1           30m
+  cattle-fleet-local-system           fleet-agent                   1/1     1            1           29m
+  cattle-fleet-system                 fleet-controller              1/1     1            1           29m
+  cattle-fleet-system                 gitjob                        1/1     1            1           29m
+  cattle-provisioning-capi-system     capi-controller-manager       1/1     1            1           28m
+  cattle-system                       rancher                       1/1     1            1           32m
+  cattle-system                       rancher-webhook               1/1     1            1           29m
+  cert-manager                        cert-manager                  1/1     1            1           32m
+  cert-manager                        cert-manager-cainjector       1/1     1            1           32m
+  cert-manager                        cert-manager-webhook          1/1     1            1           32m
+  ingress-nginx                       ingress-nginx-controller      1/1     1            1           33m
+  kube-system                         coredns                       2/2     2            2           38m
+  ```
+
+### Check Helm Chart Installation
+
+1. Check that the Helm chart installation completed:
+
+```shell
+helm ls -n cattle-rancher-csp-deployer-system
+```
+
+2. Verify the status of the installation:
+
+```shell
+helm status rancher-cloud -n cattle-rancher-csp-deployer-system
+```
+
+Refer to the [Troubleshooting](troubleshooting.md) section if installation fails.
+
+When Helm chart installation successfully completes, Rancher Prime will be installed.
+
+## Log into the Rancher Dashboard
+
+You may now log in to the Rancher dashboard by pointing your browser to the Rancher server URL `https://<host-name>`. The `<host-name>` is the hostname you entered when you [installed Rancher](#installing-rancher).
+
+:::note
+
+The Rancher hostname must be resolvable by public DNS. For more details, please refer to the [Prerequisites](prerequisites.md) section.
+
+:::
+
+## Uninstalling Rancher Prime PAYG Offering
+
+Run the following command to uninstall Rancher Prime:
+
+```shell
+helm uninstall -n cattle-rancher-csp-deployer-system rancher-cloud
+```
+
+Uninstalling Rancher Prime may not remove all of the Kubernetes resources created by Rancher. Run the [Rancher resource cleanup script](https://github.com/rancher/rancher-cleanup) to perform a more comprehensive cleanup.
+
+The best practice for uninstalling the Rancher Prime PAYG offering is to migrate any non-Rancher workloads to a different cluster and destroy the Rancher cluster.
+
+:::note warning
```

Comment:
```suggestion
:::warning
```

Warning is built-in so it doesn't need to use a custom label.

https://docusaurus.io/docs/markdown-features/admonitions